concent_api/api_testing_common.py

```diff
@@ -0,0 +1,187 @@
+import datetime
+import http.client
+import json
+import requests
+import sys
+
+from collections                    import namedtuple
+
+from golem_messages.exceptions      import MessageError
+from golem_messages.message         import ComputeTaskDef
+from golem_messages.message         import Message
+from golem_messages.message         import TaskToCompute
+from golem_messages.shortcuts       import dump
+from golem_messages.shortcuts       import load
+
+
+ProtocolConstants = namedtuple(
+    "ProtocolConstants",
+    [
+        "concent_messaging_time",
+        "subtask_verification_time",
+        "force_acceptance_time",
+        "token_expiration_time",
+    ]
+)
+
+
+def get_protocol_constants(cluster_url):
+    url                         = f"{cluster_url}/api/v1/protocol-constants/"
+    response                    = requests.get(url)
+    json                        = response.json()
+    concent_messaging_time      = json['concent_messaging_time']
+    subtask_verification_time   = json['subtask_verification_time']
+    force_acceptance_time       = json['force_acceptance_time']
+    token_expiration_time       = json['token_expiration_time']
+    constants                   = ProtocolConstants(
+            concent_messaging_time      = concent_messaging_time,
+            subtask_verification_time   = subtask_verification_time,
+            force_acceptance_time       = force_acceptance_time,
+            token_expiration_time       = token_expiration_time
+    )
+    return constants
+
+
+def print_protocol_constants(constants):
+    print("PROTOCOL_CONSTANTS: ")
+    print(f"concent_messaging_time = {constants.concent_messaging_time}")
+    print(f"subtask_verification_time = {constants.subtask_verification_time}")
+    print(f"force_acceptance_time = {constants.force_acceptance_time}")
```

Comment:
Why not do it in a loop? We won't have to update it each time we add a new constant.

Reply:
done